pkgs/applications/editors/emacs/build-support/elpa.nix

```diff
@@ -3,21 +3,24 @@
 { lib, stdenv, emacs, texinfo, writeText, gcc }:
 
 let
-  handledArgs = [ "meta" ];
   genericBuild = import ./generic.nix { inherit lib stdenv emacs texinfo writeText gcc; };
+  libBuildHelper = import ./lib-build-helper.nix { inherit lib; };
 
 in
 
+libBuildHelper.extendMkDerivation { } genericBuild (finalAttrs:
+
 { pname
 , version
 , src
```

Comment:
definition `src` is not used

Reply:
indirectly used here https://github.com/NixOS/nixpkgs/pull/330589/files#diff-73ae8b0d6ab3f94bf73f38648b4ae08138ce24ae49724306bd9aeb4daf00dbb6R36

To minimize diff, I do not change `args.src` to `src`.